frc1388-Swerve-2022/src/main/java/frc/robot/SwerveModule.java

```diff
@@ -23,11 +23,14 @@ public class SwerveModule {
     public double m_speed;
 
     private final double secPerMin = 60;
+    private final double metersPerInch = 0.0254;
     private final double motorRotationsPerWheelRotation = 6.67; // from AndyMark
     private final double inchesPerWheelRotation = 4 * Math.PI;
-    private final double metersPerInch = 0.0254;
-    private final double metersPerMotorRotation = (inchesPerWheelRotation * metersPerInch) / motorRotationsPerWheelRotation;
-    private final double metersPerSecondPerRPM = metersPerMotorRotation * secPerMin;
```

Comment:
The dimensional analysis of metersPerSecondPerRPM shows that secPerMin is inverted from what it should be.